### PR TITLE
Rewrote passName into $pass as requested

### PR DESCRIPTION
**Pull request by sashahilton00**
Also tried to write some unit tests, though my phpunit was playing up,
not sure why, hopefully it tests successfully on here…

### CODE REVIEW

**Comment by g-** [comment]
Hi Sasha, Thank you for including the tests. I see that the pass name is still being set through the constructor. I think this is problematic in scenarios where the PassFactory is packaging multiple passes. Since the pass name is always the same value, each pass would overwrite the previous one.

This could be solved by including the passName as a parameter of the package() method so that a different name can be passed in each time.

**Comment by sashahilton00** [comment]
@g Sorry about that, didn't really think about it as I'm only ever packaging single passes. HAve moved it into the package() function.

File: tests/bootstrap.php [hunk 1]
```diff
@@ -9,6 +9,6 @@
  * file that was distributed with this source code.
  */
 
-$loader = require_once __DIR__ . "/../vendor/autoload.php";
+$loader = require __DIR__ . "/../vendor/autoload.php";
```
Comment (g-) [inline tests/bootstrap.php]: What is the reason for this change? I'd expect the autoload file to only be loaded once.

File: tests/Passbook/Tests/PassTest.php [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 namespace Passbook\Tests;
 
+date_default_timezone_set('Europe/London');
```
Comment (g-) [inline tests/Passbook/Tests/PassTest.php]: Was there a problem with the test that this addresses?

File: tests/bootstrap.php (hunk 1 above)
Comment (sashahilton00) [inline tests/bootstrap.php]: The test failed when set to require once. I can't remember the exact error or the explanation for it, but this fixed it. Plus it's only in the tests that this has been changed, so not too much to worry about... I can try and reproduce the error if you want, but I am certain that it wasn't something that I later discovered that was wrong with my environment...

File: tests/Passbook/Tests/PassTest.php (hunk 2 above)
Comment (sashahilton00) [inline tests/Passbook/Tests/PassTest.php]: Yes there was, the test failed if date.timezone was not set when running the Boarding Pass test I believe. Normally this wouldn't need to be set, but I tested on a fresh php environment where I hadn't set the time and settings up, which caused the test to fail, so included this as a fix to anyone else that might have the same issue.

File: tests/Passbook/Tests/PassFactoryTest.php [hunk 3]
```diff
@@ -141,6 +141,29 @@ public function testRequiredInformationInPassNotOverwrittenByFactory()
         self::assertEquals($passTeamIdentifier, $pass->getTeamIdentifier());
         self::assertEquals($passPassTypeIdentifier, $pass->getPassTypeIdentifier());
     }
+    
+    //This is super lazy, just copied and pasted from above. Should clean up at some point...
```
Comment (g-) [inline tests/Passbook/Tests/PassFactoryTest.php]: Thank you for adding this test. I've tidied it up so the test is more concise.

File: tests/Passbook/Tests/PassTest.php (hunk 2 above)
Comment (g-) [inline tests/Passbook/Tests/PassTest.php]: Makes sense. I've moved this to the setUpBeforeClass() method.

File: tests/bootstrap.php (hunk 1 above)
Comment (g-) [inline tests/bootstrap.php]: Ok. I've taken this out of this pull request as I'm not convinced that is correct. We can address it separately if it continues to be problematic.

**Comment by sashahilton00** [comment]
Yeah I'm not sure about the require in the test, but I know require_once wasn't working for me. Assume it was just something on my setup.
